Source/DataSources/BillboardVisualizer.js

```diff
@@ -162,6 +164,35 @@ define([
     };
 
     /**
+     * Computes a bounding sphere which encloses the visualization produced for the specified entity.
+     *
+     * @param {Entity} entity The entity whose bounding sphere to compute.
+     * @param {BoundingSphere} result The bounding sphere onto which to store the result.
+     * @returns {BoundingSphereState} BoundingSphereState.DONE if the result contains the bounding sphere,
+     *                       BoundingSphereState.PENDING if the result is still being computed, or
+     *                       BoundingSphereState.FAILED if the entity has no visualization in the current scene.
+     */
+    BillboardVisualizer.prototype.getBoundingSphere = function(entity, result) {
```

Comment:
Because the `Visualizer` system as a whole is currently public and we technically support people adding their own custom visualizers.  If we wanted to do a major API scrub sometime soon, we could remove a good chunk of stuff from the public API on the assumption that many people don't need/use them, that includes all of the Visualizers and Updaters and some odds and ends.  In all it would remove about 20 classes from the public API.

The other solution would be to finally take the time to split our doc generation into different categories and put all of this kind of stuff in an `Advanced` category or something.  This would make the initial help page much smaller and use friendly.

I'm not against either of these ideas; and in both cases users can still use the functionality that's there; but if we don't do either of the above, this needs to stay public.